pydantic/fields.py

```diff
@@ -991,14 +992,26 @@ def computed_field(__func: PropertyT) -> PropertyT:
     ...
 
 
+def _wrapped_property_is_private(property_: cached_property | property) -> bool:
+    """Returns true if provided property is private, False otherwise."""
+    wrapped_name: str = ''
+
+    if isinstance(property_, property):
+        wrapped_name = getattr(property_.fget, '__name__', '')
+    elif isinstance(property_, cached_property):
+        wrapped_name = getattr(property_.func, '__name__', '')
+
+    return wrapped_name.startswith('_') and not wrapped_name.startswith('__')
+
+
 def computed_field(
     __f: PropertyT | None = None,
     *,
     alias: str | None = None,
     alias_priority: int | None = None,
     title: str | None = None,
     description: str | None = None,
-    repr: bool = True,
+    repr: bool | None = None,
```

Comment:
I think this would be a breaking change

Reply:
I still account for setting the default -> `True` within the body of the function if the field is private, so I don't think it's breaking.

Reply:
Do you mean if the field is public?

Reply:
I think this is a change for private fields. as the existing behavior seems to be a bug, so, it's good.
ignore my comment.

Reply:
@adriangb yep, I mean't public